boards/components/src/ieee802154.rs

```diff
@@ -95,10 +97,39 @@ macro_rules! ieee802154_component_static {
             >
         );
 
-        let mux_mac = kernel::static_buf!(capsules_extra::ieee802154::virtual_mac::MuxMac<'static>);
-        let mac_user =
-            kernel::static_buf!(capsules_extra::ieee802154::virtual_mac::MacUser<'static>);
-        let radio_driver = kernel::static_buf!(capsules_extra::ieee802154::RadioDriver<'static>);
+        let mux_mac = kernel::static_buf!(
+            capsules_extra::ieee802154::virtual_mac::MuxMac<
```

Comment:
This file would probably become a lot more readable with a
```rust
use capsules_extra::ieee802154::virtual_mac;
use capsules_extra::ieee802154::framer;
```
etc.,
but I don't think we need to block this PR on it.